### ISSUE TITLE
Error when "XDG_DATA_DIRS" env variable is not defined

### ISSUE THREAD

**Issue by ezio4df** [paymoapp/node-active-window#1]
When "XDG_DATA_DIRS" env variable isn't defined the app crashes with this,
```sh
terminate called after throwing an instance of 'std::logic_error'
  what():  basic_string: construction from null is not valid
zsh: IOT instruction (core dumped)  npm run demo
```

I am using Manjaro linux with KDE desktop environment and have no idea what it does or why its not defined. You can try unsetting the variable to reproduce this issue. I think some error checking should be added with helpful err msg if the env variable is really *required* for the package. 

https://github.com/paymoapp/node-active-window/blob/8a3b065a0f9e97222c6cd77746e64c18de0a9083/module/linux/src/ActiveWindow.cpp#L60

https://github.com/paymoapp/node-active-window/blob/8a3b065a0f9e97222c6cd77746e64c18de0a9083/module/linux/src/ActiveWindow.cpp#L418

I cant code C++ 😅😅, so leaving this issue

**Comment by gergof** [comment]
Hello!
Thanks for reporting the issue!

The variable itself is not _really_ required. It is used to fetch high resolution png icons for the applications.
The process is the following:
- get the desktop entries
- find the icon paths for each application entry

As a fallback we use the _NET_WM_ICON xprop, which is suitable in most of the cases, but it's not high res.

I've assumed that XDG_DATA_DIRS is set on every desktop env (it's the directory where the desktop entries and the icons should be stored), but it seems like it's not. :)
I will add a check that the env var is set, if not, it will fall back to _NET_WM_ICON.

**Comment by gergof** [comment]
Fixed in 42d165567ec84304ac495e7c798d52085dad2a53
A build is on it's way!
